canonical-web-and-design/ubuntu.com
# Reactify microcerts header

## Done

- Converted the header on the microcerts page to render with React

## QA

- Run the server with `dotrun`
- Go to http://localhost:8001/cube/microcerts?test_backend=true
- Verify that the header is displayed and corresponds to the badges you've received

## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/369

## Screenshots

### In progress
![microcerts-header](https://user-images.githubusercontent.com/995051/143863747-50d25d02-64ea-4df8-ad08-d0e7fb6f60a1.png)

### Complete
![microcerts-header-complete](https://user-images.githubusercontent.com/995051/143865047-e760599a-5573-4f00-895c-23b7de5fe41e.png)
